resources/observability/prometheusrules/observatorium-custom-metrics-production.prometheusrules.yaml

```
@@ -13,7 +13,7 @@ spec:
     rules:
     - alert: ObservatoriumNoStoreBlocksLoaded
       annotations:
-        dashboard: https://grafana.app-sre.devshift.net/d/no-dashboard/observatorium-metrics?orgId=1&refresh=10s&var-datasource={{$labels.cluster}}-prometheus&var-namespace={{$labels.namespace}}&var-job=All&var-pod=All&var-interval=5m
+        dashboard: https://grafana.app-sre.devshift.net/d/no-dashboard/observatorium-metrics?orgId=1&refresh=10s&var-namespace={{$labels.namespace}}&var-job=All&var-pod=All&var-interval=5m
```

Why don' we just use prometheus externalLabels here: https://prometheus.telemeter-prod-01.devshift.net/config or https://prometheus.telemeter-prod-01.devshift.net/config?
```suggestion
        dashboard: https://grafana.app-sre.devshift.net/d/no-dashboard/observatorium-metrics?orgId=1&refresh=10s&var-datasource={{$externalLabels.cluster}}-prometheus&var-job=All&var-pod=All&var-interval=5m
```